src/Entity/CommerceContentEntityBase.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Drupal\commerce\Entity;
+
+use Drupal\Core\Entity\ContentEntityBase;
+use Drupal\Core\Language\LanguageInterface;
+
+/**
+ * Provides the base class for Commerce content entities.
+ */
+class CommerceContentEntityBase extends ContentEntityBase implements CommerceContentEntityInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getTranslatedReferencedEntities($field_name) {
+    $refereced_entities = $this->get($field_name)->referencedEntities();
+    return $this->ensureTranslations($refereced_entities);
```

Comment:
typo error on $refereced_entities. I guess you miss a 'n' in $referenced_entities